src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -488,7 +488,7 @@ class Snapshot(_AnsiblePlaybookAlias):
     help = (
         "This command creates a snapshot of your current environment's state.\n\n"
         "State information is saved in the '~/.commcare-cloud/snapshots' directory. It is a good idea to run this"
-        "before making any major changes to your environment, as it allows you to have a record of your"
+        " before making any major changes to your environment, as it allows you to have a record of your"
```

Comment:
is this needed? If yes, I guess better to have an empty space at the end of the earlier line.
Looks like you have the same thing now for the next line, `yourenvironment's` or for `run thisbefore` in the line above this

Reply:
Good call. I updated the code :+1: